applications/ParticleMechanicsApplication/python_scripts/apply_mpm_slip_boundary_process.py

```diff
@@ -1,5 +1,5 @@
 import KratosMultiphysics
-import KratosMultiphysics.ParticleMechanicsApplication
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
```

Comment:
why do you change this? This is not related to the python modules

Reply:
This is just a minor cleanup, to make all the import called as KratosParticle. Not related yes.

Reply:
I checked the file, you are not using anything from the ParticlesApp there, therfore I recommend removing them (also codacy complains abt it being unused which is true)

I recommend to only import them if you need them

Reply:
we can discuss in person if you have questions

Reply:
I agree with you. In fact, I removed it from some other python files too. Thanks for the suggestion!

Reply:
It is not correct yet, I will explain it to you in person tmr
Nothing big, don't worry